webapp/src/theme.ts

```diff
@@ -2,6 +2,9 @@
 // See LICENSE.txt for license information.
 
 import {CSSObject} from '@emotion/serialize'
+import * as _ from 'lodash'
```

Comment:
Also, I think this imports everything here, so the bundle size is bigger. Basically we are including all the lodash library to use 1 function. A better approach would be something like:

```js
import isEqual from 'lodash/isEqual'
```

this way is only importing the isEqual code, and the bundle size should be slightly smaller.

Reply:
Done.